webgme-setup.json

```diff
@@ -12,6 +12,13 @@
       "ValidateKeras": {
         "src": "src/plugins/ValidateKeras",
         "test": "test/plugins/ValidateKeras"
+<<<<<<< HEAD
```

Comment:
This merge conflict should be resolved and not just committed. It should have both entries in the webgme-setup.json (and be valid JSON) then run `webgme refresh`